rosidl_generator_cpp/msg/PrimitivesConstants.msg

```diff
@@ -0,0 +1,14 @@
+bool BOOL_CONST=true
+byte BYTE_CONST=50
+char CHAR_CONST=100
+float32 FLOAT32_CONST=1.125
+float64 FLOAT64_CONST=3.14159
```

Comment:
But this line uses `3.14159` and not was we did in the reference patch to test_communication.